app/view_models/v1/work_item.rb

```diff
@@ -68,9 +68,9 @@ def table_fields
       [
         work_type.to_s,
         "#{provider_requested_uplift.to_i}%",
-        NumberTo.pounds(provider_requested_amount).to_s,
+        ApplicationController.helpers.format_period(provider_requested_time_spent, style: :long),
```

Comment:
🤔 i wonder if there is a way to mixin these helpers, for brevity if nothing else but no biggee

Reply:
there is, and I have considered it previously but we only had a few instances of doing it so it didn;t seem value for money... that being said it may now be time to re-evaluate that decision...

might look to do that in a separate PR. :)